### PR TITLE
First step of startup refactoring

### PR DESCRIPTION
**Pull request by svenreimers**
Adresses #162 
-removes blocking stuff
- more lazy initilaization

### CODE REVIEW

**Comment by mklaehn** [comment]
@svenreimers: The newly added files from the third commit (643613e) should not have been added to the PR

File: core/src/main/java/org/tweetwallfx/tweet/TweetSetData.java [hunk 1]
```diff
@@ -23,41 +23,34 @@
...
 
-    public TweetStream getTweetStream() {
+    public synchronized TweetStream getTweetStream() {
+        if (null == tweetStream) {
+            tweetStream = getTweeter().createTweetStream(query);
+        }
         return tweetStream;
     }
 
     public Tweeter getTweeter() {
```
Comment (mklaehn) [inline core/src/main/java/org/tweetwallfx/tweet/TweetSetData.java]: consider removing this method. Tweeter can be accesed via static singleton method

File: devoxx-2017-be/src/main/resources/config.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+tweetwall.twitter.query=#devoxx OR #devoxxus
```
Comment (mklaehn) [inline devoxx-2017-be/src/main/resources/config.properties]: this file does not belong into the PR

File: devoxx-2017-be/src/main/resources/log4j2.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mklaehn) [inline devoxx-2017-be/src/main/resources/log4j2.xml]: this file does not belong into the PR

File: devoxx-2017-be/src/main/resources/tweetwall.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
+/*
```
Comment (mklaehn) [inline devoxx-2017-be/src/main/resources/tweetwall.css]: this file does not belong into the PR

**Comment by svenreimers** [comment]
Ok you are correct will rework.

**Pull request merged**